packages/vhd-lib/src/createVhdStreamWithLength.integ.spec.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-env jest */
+
+import execa from 'execa'
+import fs from 'fs-extra'
+import rimraf from 'rimraf'
+import tmp from 'tmp'
+import { createReadStream, createWriteStream } from 'fs-promise'
+import { fromEvent, pFromCallback } from 'promise-toolbox'
+import { Slicer } from 'pipette'
+
+import { createVhdStreamWithLength } from '.'
+import { FOOTER_SIZE } from './_constants'
+
+const initialDir = process.cwd()
+
+beforeEach(async () => {
+  const dir = await pFromCallback(cb => tmp.dir(cb))
+  process.chdir(dir)
```

Comment:
Changing the working directory seems to be a bad idea, wouldn't it be possible to simply use this path as a prefix?

Reply:
I think it would work here, do you want me to also change it in all the other tests?

Reply:
No, let's focus on this new code :slightly_smiling_face: